unstructured_inference/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.8.7"  # pragma: no cover
+__version__ = "0.8.8-dev1"  # pragma: no cover
```

Comment:
This can stay as it is, but just an fyi that we start the dev versions with `dev0`